jina/__init__.py

```diff
@@ -125,18 +125,18 @@ def _set_nofile(nofile_atleast=4096):
 # ONLY FIRST CLASS CITIZENS ARE ALLOWED HERE, namely Document, Executor Flow
 
 # Document
-from docarray import Document, DocumentArray
+from docarray.documents.legacy import Document, DocumentArray
```

Comment:
why is `DocumentArray` needed to be imported from legacy?

Reply:
good point should be from docarray directly